modules/order/order.graphql.js

```diff
@@ -29,21 +39,17 @@ enum Status {
   SENT
   DELIVERED
 }
-type OrderItems {
-  category: [Language]
-  subcategory: [Language]
-  model: [Language]
-  name: [Language]
-  colors: [[Language]]
-  pattern: [Language]
-  closure: [Language]
-  closureColor: String
-  size: Size,
-  bottomMaterial: [Language]
-  bottomColor: [Language]
-  additions: [[Language]]
-  actualPrice: [CurrencySet]
+type OrderItem {
+  product: Product
+  model: Model
   quantity: Int
+  isFromConstructor: Boolean
+  options: ItemOptions
+  constructorBasic: ConstructorBasic
+  constructorPattern: Pattern
+  constructorFrontPocket: ConstructorFrontPocket
+  constructorBottom: ConstructorBottom
+  fixedPrice: [CurrencySet]
 }
 type Delivery {
```

Comment:
You need to have `enum` on `sendBy` field on this type

Reply:
fixed